dali/operators/sequence/optical_flow/optical_flow.h

```diff
@@ -145,18 +145,11 @@ class OpticalFlow : public StatelessOperator<Backend> {
    */
   void of_lazy_init(size_t width, size_t height, size_t channels, DALIImageType image_type,
                     int device_id, cudaStream_t stream) {
-    std::call_once(of_initialized_,
-                   [&]() {
-                       optical_flow_.reset(
-                               new optical_flow::OpticalFlowImpl(of_params_,
-                                                                 width,
-                                                                 height,
-                                                                 channels,
-                                                                 image_type,
-                                                                 device_id,
-                                                                 stream));
-                        optical_flow_->Init(of_params_);
-                   });
```

Comment:
This is called from within setup. No atomicity is required, setup is never called in parallel with itself.